### PR TITLE
2024-02 newsletter

### PR DESCRIPTION
**Pull request by ptheywood**
Adds Feb 2024 newsletter.

### CODE REVIEW

File: _newsletters/2024-02-14-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+tags:
+category:
+link:
+description:
+type: text
+---
+
+## The University of Sheffield Research Software Engineering Community Newsletter February 2024
+
+Welcome to the January 2024 newsletter for the research software community at The University of Sheffield, featuring
```
Comment (tdjames1) [inline _newsletters/2024-02-14-newsletter.md]:
```suggestion
Welcome to the February 2024 newsletter for the research software community at The University of Sheffield, featuring
```

**Comment by twinkarma** [comment]
@ptheywood Can we advertise the upcoming DL course please!

Here's the link 
https://mydevelopment.csod.com/ui/lms-learning-details/app/event/8fb8273d-99ab-4e3e-8ce9-65242c2215ba

Date: 1-5pm on 22nd and 23rd Feb

Title:
Deep Learning Demystified: Foundations for Non-Computer Scientist

Description:
"Deep Learning Demystified: Foundations for Non-Computer Scientists" is an accessible and comprehensive course designed to introduce individuals from diverse backgrounds to the fundamental concepts of deep learning. Through clear explanations and real-world examples, participants will gain a solid understanding of key components of deep learning. By the end of the course, students will be equipped with the knowledge and confidence to engage with and apply deep learning techniques in various fields, regardless of their technical background.

File: _newsletters/2024-02-14-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  -[Website](https://www.reprohack.org/event/27/)
+- RSE Midlands meeting in Warwick on 2024-03-07
+  - The event is fully booked, but a wait list is available.
+  - [Website](https://rse-midlands.github.io/docs/event-7th-march-2024/)
+  - [Registration (Wait list)](https://pretix.eu/rsemidlands/2024/)
+- Carpentry Connect Heidelberg 2024
+  - 2024-11-12 to 2024-11-14, Heidelberg, Germany
+  - [Website](https://carpentries.org/blog/2024/01/announcing-cchd24/)
+- Royal Statistical Society Talk: - Medical Statistician Apprenticeships and Integrating R into GXP Medical Research
+  - 2024-02-28 1600-17:30
```
Comment (tdjames1) [inline _newsletters/2024-02-14-newsletter.md]:
```suggestion
  - 2024-02-28 16:00-17:30
```

File: _newsletters/2024-02-14-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  - [Website](https://user2024.r-project.org)
+- An introduction to [Octopus.ac](https://www.octopus.ac/)
+  - 2024-03-28 Online
+  - Octopus.ac is a free, UKRI funded publishing model that forgoes the traditional research paper in favour of a series of “micro publications” that more closely model how research is conducted.
+  During this session we’ll invite you to share your perceptions of Octopus and welcome feedback from researchers of all disciplines
+  - [Event Registration](https://events.teams.microsoft.com/event/f6858265-3a51-4dac-b6bd-09e478c3cd2d@48f9394d-8a14-4d27-82a6-f35f12361205)
+
+### Open Research
+
+- FAIR Faculty seminar series is now all scheduled on my development (requires login)
```
Comment (tdjames1) [inline _newsletters/2024-02-14-newsletter.md]:
```suggestion
- FAIR Faculty seminar series is now all scheduled on myDevelopment (requires login)
```

File: _newsletters/2024-02-14-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  - Various dates in February, grouped by Faculty.
+  - [Mydevelopment](https://mydevelopment.csod.com/ui/lms-learner-search/search?pageNumber=1&query=FAIR%20faculty%20seminars)
```
Comment (tdjames1) [inline _newsletters/2024-02-14-newsletter.md]:
```suggestion
  - [myDevelopment](https://mydevelopment.csod.com/ui/lms-learner-search/search?pageNumber=1&query=FAIR%20faculty%20seminars)
```

**Review by tdjames1 — CHANGES_REQUESTED** [review]
Looks good, just some minor edits suggested.

**Comment by ptheywood** [comment]
Thanks @tdjames1, I've committed your suggetions. 

@twinkarma I've added the deep learning demistifyed course to the Training section in 102f0ba

**Comment by twinkarma** [comment]
Thanks @ptheywood, the rest of the contents looks good to me as well!

File: _newsletters/2024-02-14-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,245 @@
...
+- [Code for Thought: [EN] Patterns in Science And Medicine - Veronika Cheplygina](https://codeforthought.buzzsprout.com/1326658/14338771-en-patterns-in-science-and-medicine-veronika-cheplygina)
+- [Code for Thought: [EN] ByteSized RSE: Easy Does It - with easybuild - Jörg Saßmannshausen](https://codeforthought.buzzsprout.com/1326658/14361557-en-bytesized-rse-easy-does-it-with-easybuild-jorg-sassmannshausen)
+
+<!-- ### Tutorial of the month -->
+
+### Training
+
+#### Deep Learning Demystified: Foundations for Non-Computer Scientists
+
+- 2024-02-22 13:00-17:00 and 2024-02-32 13:00-17:00
```
Comment (tdjames1) [inline _newsletters/2024-02-14-newsletter.md]:
```suggestion
- 2024-02-22 13:00-17:00 and 2024-02-23 13:00-17:00
```

Comment (tdjames1) [reply _newsletters/2024-02-14-newsletter.md]: One more typo!

**Review by tdjames1 — CHANGES_REQUESTED** [review]

File: _newsletters/2024-02-14-newsletter.md (hunk 5 above)
Comment (ptheywood) [reply _newsletters/2024-02-14-newsletter.md]: ta

**Review by tdjames1 — APPROVED** [review]

**Pull request merged**